errai-cdi/jboss/src/main/java/org/jboss/errai/cdi/server/gwt/JBossLauncher.java

```diff
@@ -38,6 +38,10 @@
   private final String CLASS_HIDING_JAVA_AGENT_PROPERTY = "errai.jboss.javaagent.path";
   private final String JBOSS_JAVA_OPTS_PROPERTY = "errai.jboss.javaopts";
   private final String TMP_CONFIG_FILE = "standalone-errai-dev.xml";
+  
+  // Added 02/13/2016
```

Comment:
Was this comment added by your IDE? It would be good to remove since it is possible to find when these fields were added through Git.